src/main/java/com/ays/auth/service/impl/AysTokenServiceImpl.java

```diff
@@ -96,7 +96,7 @@ public AysToken generate(final Map<String, Object> claims, final String refreshT
                 .setIssuer(tokenConfiguration.getIssuer())
                 .setIssuedAt(accessTokenIssuedAt)
                 .setExpiration(accessTokenExpiresAt)
-                .claim(AysTokenClaims.TYPE.getValue(), OAuth2AccessToken.TokenType.BEARER.getValue())
+                .setHeaderParam(AysTokenClaims.TYPE.getValue(), OAuth2AccessToken.TokenType.BEARER.getValue())
                 .signWith(tokenConfiguration.getPrivateKey(), SignatureAlgorithm.RS512)
                 .addClaims(claims)
                 .compact();
```

Comment:
Commit etmeden önce kodu formatlayabilir miyiz? Boşluklar bu sınıf için doğru düzenlenmemiş görünüyor.